2019-11-20 12:42:19.220 - [Info] MITM Version: 1.26
2019-11-20 12:42:19.226 - [Info] Auto Access Enabled: true
2019-11-20 12:42:19.227 - [Debug] [Init] Auto Access Barrier: true
2019-11-20 12:42:19.250 - [Info] SSH man-in-the-middle server for 172.20.0.6 listening on 0.0.0.0:10105
2019-11-20 13:08:31.308 - [Debug] [Connection] Attacker connected: 106.13.219.171 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 13:08:33.489 - [Debug] [Auth] Attacker 106.13.219.171 trying to authenticate with "password"
2019-11-20 13:08:33.491 - [Debug] [Auto Access] Attacker: 106.13.219.171, Threshold: 1, Attempts: 1
2019-11-20 13:08:33.491 - [Debug] [Auto Access] Compromising the honeypot
2019-11-20 13:08:33.491 - [Debug] [Auto Access] Adding the following credentials: "mysql:serenity"
2019-11-20 13:08:33.882 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-20 13:08:33.882 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.6
2019-11-20 13:08:33.942 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-20 13:08:35.008 - [Debug] [Connection] Attacker connected: 106.13.219.171 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 13:08:36.952 - [Debug] [Auth] Attacker 106.13.219.171 trying to authenticate with "password"
2019-11-20 13:08:36.954 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-20 13:08:36.956 - [Debug] [Auth] Attacker: 106.13.219.171 has so far made 1 attempts. Remaining: 5 attempts
2019-11-20 13:08:37.512 - [Debug] [Connection] Attacker closed the connection
2019-11-20 13:08:38.015 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-20 13:14:46.380 - [Debug] [Connection] Client error on ssh server
2019-11-20 13:14:46.381 - [Debug] [Connection] Attacker closed connection
2019-11-20 13:14:46.388 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-20 13:14:46.388 - [Debug] [LXC] Container's OpenSSH server closed connection
